cleanlab/filter.py

```diff
@@ -754,7 +768,7 @@ def _prune_by_count(k, args=None) -> np.ndarray:
     return label_issues_mask
 
 
-def _multiclass_crossval_predict(labels, pyx) -> np.ndarray:
+def _multiclass_crossval_predict(labels: List[List[str]], pyx: np.ndarray) -> np.ndarray:
```

Comment:
For now, let's just focus on the annotations:

```suggestion
def _multiclass_crossval_predict(labels: List[List[int]], pyx: np.ndarray) -> np.ndarray:
```

Reply:
This makes it a lot easier, at the start I didn't even know what label actually was.